### ISSUE TITLE
Required go version

### ISSUE THREAD

**Issue by etam** [trezor/trezord-go#283]
README.md says that "trezord-go requires go >= 1.12". But at https://github.com/trezor/trezord-go/blob/v2.0.32/trezord.go#L105 BuildInfo.Settings is used, which according to documentation (https://pkg.go.dev/runtime/debug#BuildSetting) was added in go 1.18.

**Comment by prusnak** [comment]
The whole sentence reads `trezord-go requires go >= 1.12, but 1.18 is recommended for git version stamping (and for better speed in general)`

Are you implying the code above does not build on Go 1.12?

**Comment by etam** [comment]
with go 1.16 I get this error:

```
./trezord.go:105:24: info.Settings undefined (type *debug.BuildInfo has no field or method Settings)
```

**Comment by prusnak** [comment]
I confirm I see the same error with Go 1.17.

Readme fixed in f559ee5079679aeb5f897c65318d3310f78223ca

Thanks!
